x-pack/plugins/ingest_manager/common/services/config_to_yaml.ts

```diff
@@ -16,7 +16,7 @@ const CONFIG_KEYS_ORDER = [
   'inputs',
   'enabled',
   'use_output',
-  'package',
+  'meta',
```

Comment:
@michalpristas for your awareness. As Agent ignores this at the moment, it should not have any effect.